src/site/_data/site.js

```diff
@@ -2,15 +2,18 @@ const getPagesData = require('../../../lib/get-pages-data')
 const getNavigationData = require('../../../lib/get-navigation-data')
 const Storyblok = require('../../lib/storyblok-instance')
 
+const events = require('./events')
+
 module.exports = async () => {
   const env = process.env.ELEVENTY_ENV
   const version = env === 'production' ? 'published' : 'draft'
 
   const result = await Storyblok.get('cdn/stories', { version })
   const { stories = [] } = result.data
+  const eventData = await events() // Because of the way eleventy works, I fetch the event data once more to pass on to components that have events. Because this is the only place where you can async request events.
 
   return {
     navigation: getNavigationData(stories),
-    stories: getPagesData(stories)
+    stories: getPagesData(stories, eventData)
```

Comment:
Hierdoor wordt de data van de events gebonden aan de pagina's, dit maakt dat logica overal vloeit en maakt alles heel onoverzichtelijk en arbitrair. Ik weet niet welk probleem je probeert op te lossen dus laten we het hier morgen even over hebben.